korapay/Foodies-Haven
# BEI-6: Setup linting with Eslint

#### WHY THIS PR?
- To auto-check common errors and maintain consistent code style.

#### WHAT DOES THIS PR DO?
- It helps to catch common mistakes.

- It helps to maintain a consistent code style.

- It also improves code readability and maintainability.

#### HOW CAN THIS PR BE TESTED?
- Create some test JavaScript or TypeScript files in your project or use existing files and
ensure that these files contain various scenarios and coding patterns that you want to test with ESLint.

#### THINGS YOU MAY WANT THE REVIEWER TO NOTE.
- Enable ESLint style checking.
